pom.xml

```diff
@@ -83,6 +83,11 @@
         -->
         <dependencies>
             <!-- viewer-->
+            <dependency>
+                <groupId>nl.opengeogroep</groupId>
+                <artifactId>safetymaps-api</artifactId>
+                <version>1.0-SNAPSHOT</version>
```

Comment:
er moet een release versie in dependencyManagement gebruikt worden, anders kun je geen release van TM maken